include/luabind/wrapper.hpp

```diff
@@ -162,6 +162,40 @@ struct function_wrapper<R (*)(Args...), func> {
     }
 };
 
+template <typename F, F f>
+struct static_function_wrapper;
+
+template <typename R, typename... Args, R (*func)(Args...)>
+struct static_function_wrapper<R (*)(Args...), func> {
```

Comment:
Why this class was added? the function_wrapper covers this case fully.

Reply:
function_wrapper has a check that the argument count should be sizeof...(Args) but for class functions it should be sizeof...(Args) + 1, as user will call this functions as Class:func(args)